src/XRP/issued-currency-client.ts

```diff
@@ -168,7 +168,6 @@ export default class IssuedCurrencyClient {
   public async disallowIncomingXrp(wallet: Wallet): Promise<TransactionResult> {
     return this.changeFlag(AccountSetFlag.asfDisallowXRP, true, wallet)
   }
-
```

Comment:
did we want to get rid of this line?

Reply:
Nope, probably an accident during merge conflict resolution